backend/question-service/server/app/readme.md

```diff
@@ -0,0 +1,321 @@
+# Question Service Server
+
+Question Service server used to manage questions for the Question Service. This service is used to create, read, update, and delete questions.
+
+## File Structure
+
+```bash
+    app/
+├── server.py # Main file to run the server
+├── populate_db.py # Script to populate the database with questions
+├── entrypoint.sh # Entrypoint script for Docker
+├── requirements.txt # Python dependencies
+├── core/
+│   ├── __init__.py
+│   ├── database.py # Database connection functions
+├── routers/
+    ├── __init__.py
+    ├── questions.py # Question endpoints
+    ├── admin.py # Admin endpoints
+├── utils/
+    ├── __init__.py
+    ├── exception_handler.py # Custom HTTP exception handler
+    ├── jsend.py # JSend response builder
+├── data/
+│   ├── BASE_QUESTIONS.json # Base questions to populate the database with
+```
+
+## Setup
+
+Ensure you have the following installed:
+
+- [Python 3.9+](https://www.python.org/downloads/)
+
+### Environment Variables
+
+Create a `.env.development` file in the root directory of the project. An example `.env.example` has been provided for you.
+
+### Install Dependencies
+
+```bash
+pip install -r requirements.txt
+```
+
+## Running the Server
+
+To run the server standalone:
+
+```bash
+uvicorn server:app --host 0.0.0.0 --port 8000
+```
+
+To run the server with Docker:
+
+```bash
+docker build -t question-service .
+docker run -p 8000:8000 question-service
```

Comment:
```suggestion
docker run -p 8000:80 question-service
```

shouldn't it be `8000:80` instead of `8000:8000`? since the Dockerfile uses the `entrypoint.sh` which uses port `80`

Reply:
Resolved in 6617a42ae09dab7ee6beaa7d478b25451f56c033